LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Extensions/Font+NetworkProtection.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  Font+NetworkProtection.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+
+extension Font {
```

Comment:
Sharing fonts so they can be used across NetP views.

Reply:
I was going to suggest that [DesignResourcesKit](https://github.com/duckduckgo/DesignResourcesKit) may be useful here, but I realized it isn't used on macOS. Never mind :)